<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about</class>
 <widget class="QDialog" name="about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/icon/icon/write.png</normaloff>:/new/icon/icon/write.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>266</width>
     <height>118</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;开发者&lt;/span&gt;：Fahy&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;联系方式&lt;/span&gt;：&lt;/p&gt;&lt;p&gt;    QQ：2195817092&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;开发时间：&lt;/span&gt;2016-01-08&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;备注&lt;/span&gt;：开发自用，源码公开，有需要，请联系作者&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="movieLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>77</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="stopPushButton">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startPushButton">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>movieLabel</zorder>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
